tests/test_estimation.py

```diff
@@ -10,6 +10,9 @@
 
 cdnow_customers = load_cdnow()
 cdnow_customers_with_monetary_value = load_summary_data_with_monetary_value()
+returning_cdnow_customers_with_monetary_value = cdnow_customers_with_monetary_value[
+    cdnow_customers_with_monetary_value['frequency'] > 0
+]
```

Comment:
For `TestGammaGammaFitter.test_params_out_is_close_to_Hardie_paper`. I can move this variable to that test if you prefer. The problem is that `cdnow_customers_with_monetary_value` contains many customers with monetary value of zero, so the new check fails. Fitting the model only on returning customers matches the example from the readme (https://github.com/CamDavidsonPilon/lifetimes#estimating-customers-lifetime-value).